tests/rptest/tests/controller_upgrade_test.py

```diff
@@ -30,12 +25,11 @@ def test_updating_cluster_when_executing_operations(self):
         Validates that cluster is operational when upgrading controller log
         '''
 
-        # set redpanda logical version to value without __consumer_offsets support
-        self.redpanda = RedpandaService(
-            self.test_context,
-            5,
-            # set redpanda logical version to v22.1 - last version without serde encoding
-            environment={"__REDPANDA_LOGICAL_VERSION": 3})
+        # set redpanda version to v22.1 - last version without serde encoding
+        self.redpanda = RedpandaService(self.test_context, 5)
+
+        installer = self.redpanda._installer
+        installer.install(self.redpanda.nodes, (22, 1, 4))
```

Comment:
I don't think so. The latest version without __consumer_offsets was 21.11.x, which doesn't play well with the admin op fuzzer because it exercises config updates, which differs in newer versions.